src/main/webapp/WEB-INF/cron.xml

```diff
@@ -12,6 +12,12 @@
       <timezone>Australia/Perth</timezone>
     </cron>
     <cron>
+      <url>/feedbackSessionClosedReminders</url>
+      <description>Checks for feedback sessions that are closed and send email to all the instructors of that course.</description>
+      <schedule>every 60 minutes from 00:06 to 23:59</schedule>
+      <timezone>Australia/Perth</timezone>
+    </cron>
+    <cron>
```

Comment:
Yeah, my bad. I assumed we had a cron job. As it turns out, we don't actually 'close' sessions, so there is no cron job for it.